app/views/components/_download-link.html.erb

```diff
@@ -0,0 +1,9 @@
+<%
+  download_link_text ||= "Download File"
+%>
+
+<div class="app-c-download-link">
```

Comment:
Is the wrapper needed to ensure clearing is done correctly? I'm not sure, simply asking. We can't make the link block level because the focus state would look weirdly wide.